tests/chart_tests/test_astronomer_cp_dp_feature.py

```diff
@@ -3,6 +3,19 @@
 from tests import supported_k8s_versions
 from tests.chart_tests.helm_template_generator import render_chart
 
+# Some Values would be added later to these maps
+# "astronomer-jetstream": "dataplane",
+# "astronomer-prometheus": "controlplane",
+COMPONENT_PLANE_MAP = {
```

Comment:
This data structure doesn't appear to handle components that will exist in both planes, like prometheus.

Reply:
what i mean is by having this be a dict, we can't put one component in two planes. EG: this is invalid:
```py
COMPONENT_PLANE_MAP = {
  "astronomer-prometheus": "controlplane",
  "astronomer-prometheus": "dataplane",
}
```

Reply:
This also exposes that having the "plane" label be a single field is insufficient.